features/text-stroke-fill.yml

```
@@ -0,0 +1,10 @@
+name: Text stroke and fill
```

```suggestion
name: Text stroke and fill  (compatibility prefixes)
```
This mirrors the `resolution-compat` and `supports-compat` features. Those are a bit different since there are recommended alternatives, but I think the qualifier still applies.